ansible/roles/csm.storage.smartmon/tasks/main.yml

```diff
@@ -37,8 +37,13 @@
 - name: Redeploy node-exporter
   when:
     - admin_keyring.stat.exists
+  command: "cephadm shell --mount /etc/cray/ceph/ -- ceph orch apply -i /mnt/node-exporter.yml"
```

Comment:
Would it be safer to execute all of the commands through the cephadm shell?

Reply:
No, the other commands should work on all systems. Applying the node-exporter configuration is the only part that could that could cause a problem based on the ceph-common version.